packages/client/src/index.ts

```diff
@@ -35,27 +37,53 @@ export const appendSlashIfNeeded = (url: string): string => {
 export default class LogtoClient {
   private readonly oidcConfiguration: OIDCConfiguration;
   private readonly clientId: string;
+  private readonly scope: string;
   private readonly sessionManager: SessionManager;
   private readonly storage: ClientStorage;
   private tokenSet: Optional<TokenSet>;
   constructor(config: ConfigParameters, oidcConfiguration: OIDCConfiguration) {
-    const { clientId, storage } = config;
+    const { clientId, storage, scope } = config;
     this.clientId = clientId;
+    this.scope = LogtoClient.correctScope(scope);
     this.oidcConfiguration = oidcConfiguration;
     this.storage = storage ?? new LocalStorage();
     this.sessionManager = new SessionManager(this.storage);
     this.createTokenSetFromCache();
   }
 
+  static correctScope(originalScope: string | string[]): string {
```

Comment:
put it into utils? no need to be public

Reply:
could align with auth0 only accepts string with empty space split the scopes

Reply:
can we could have the trim check on the original scope input param

Reply:
i prefer `string | string[]`